### ISSUE TITLE
implement genetic operators to achieve modification

### ISSUE THREAD

**Issue by mcguenther** [digital-bauhaus/thor-2#7]
as of now, the user describes the modifications to be applied to an existing AVM for the modification task.
the modifications are applied to the AVM to generate the initial population for the genetic algorithm, which finishes after 3-4 iterations. I would argue that in the genetic algorithm, no substantial changes happen, hence the quick termination.

in contrast, we might init the population with duplicates of the original AVM and achieve the requirements of the user by applying not only crossovers, but also mutations; therefor, we would need to adapt both the genetic operations and the fitness function.
As a result, one might excpect to find a result which meets the desired changes not exactly but is more authentic.

**Comment by mcguenther** [comment]
#2 should be done before tackling this issue
